### DISCUSSION TITLE
feature: Allow multiple/all locales for isAlpha and IsAlphanumeric decorator

### DISCUSSION THREAD

**Discussion by Alfagun74** [typestack/class-validator#2366] [Feature Ideas]
### Description
I validate the Name Field using `IsAlpha()`. German users need Umlauts (ä,ö,ü, etc.), so I've set it to `IsAlpha("de-DE")`. Now, Spanish users are requesting (ó,ú,á), and tomorrow, Turkish users may request their special characters. I need a solution for this.

### Proposed Solution
We can consider using an array of locales to allow, for example, `IsAlpha(["de", "es"])`, or a more inclusive approach, such as allowing all locales with `IsAlpha("all")`.

### Question
Is there a better decorator for handling names? What if the name is Arabic or Chinese, and not Latin at all? I'm quite curious.

Related: #266

**Comment by Z4nR** [comment]
I thing that is a great idea, so some app or framework using class-validator and have a global user don't need to write every naming styling in decorator

**Comment by braaar** [comment]
I'm moving this to the discussion tab, as we are trying to keep the issue count low. Please see [this issue](https://github.com/typestack/class-validator/issues/1775) for an explanation.

**Reply by Alfagun74** [reply]
this sure is one way to solve issues. damn.
